### DISCUSSION TITLE
user profile avatar and header image urls using 127.0.0.1 instead of my fqdn

### DISCUSSION THREAD

**Discussion by rmarchant** [mastodon/mastodon#22387] [Q&A]
Anyone experience something similar?

http://127.0.0.1/system/accounts/avatars/109/526/357/113/823/465/original/6c6daa0f7a65632f.png

Fresh install on a headless server, so never accessed the website via localhost, only the fqdn that's set as the LOCAL_DOMAIN.

This occurs on my profile page and posts. 

Interestingly, on the 'edit profile appearance' page, my avatar and header image is properly shown using my fqdn.

But the link to my profile that's wrapped around the two images on the 'profile appearance page' is: 

http://127.0.0.1/@my_user

However, the link to my profile on the left nav bar uses the fqdn.

Is it possible to edit this value somewhere?

**Comment by mjankowski** [comment]
Did you ever work this out?

**Reply by rmarchant** [reply]
Unfortunately not. No longer self-hosting this service.

**Reply by mjankowski** [reply]
Got it, thanks.
